CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+- Switch metrics to use new JSON-based data store. ([#1470](https://github.com/heroku/heroku-buildpack-nodejs/pull/1470))
+
+## [v306] - 2025-08-21
+
+- Rollback to v304 functionality due to increase in `bin/report` failures.
```

Comment:
(optional) As david mentioned this might create some unexpected automation issues: Another option would be to annotate inside of v305 somehow that the changes were rolled back instead of adding a new deploy version. I don't have a preference one way or the other, but noting it as an option.

Reply:
I think @schneems means branch off tag v305 and put that line in there, but tagging that as v306 would give you a tag of the code of v305, which isn't what we'd want.

If you're bored, you could branch off of v304, add the v305 changes and the v306 revert remark, and tag that as v306 ;)